script/deploy

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ruby
```

Comment:
Python has [`optparse`](https://docs.python.org/3/library/optparse.html) and [`argparse`](https://docs.python.org/3/library/argparse.html?highlight=argparse#module-argparse)… it's kinda spoiled for choice. There's also [`docopt`](https://github.com/docopt/docopt) and [`click`](http://click.pocoo.org/6/) (which I'm kinda partial to).